tests/test_fits_to_zarr.py

```
@@ -125,10 +135,417 @@ def test_mhz_from_name_no_match():
+        coords={
+            "l": ("l", l_sm),
+            "m": ("m", m_sm),
+            "right_ascension": (("m", "l"), np.zeros((3, 4))),
+            "declination": (("m", "l"), np.zeros((3, 4))),
+        },
+        attrs={"fits_wcs_header": hdr_ref},
+    )
+
+    with (
+        patch.object(xr.Dataset, "interp", side_effect=ValueError("simulated interp failure")),
+        pytest.raises(RuntimeError, match="bad_file.fits"),
```

Ruff (0.15.10)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)